Firestore/core/src/core/not_in_filter.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIRESTORE_CORE_SRC_CORE_NOT_IN_FILTER_H_
+#define FIRESTORE_CORE_SRC_CORE_NOT_IN_FILTER_H_
+
+#include <string>
+
+#include "Firestore/core/src/core/field_filter.h"
+
+namespace firebase {
+namespace firestore {
+
+namespace model {
```

Comment:
You can get these forward declarations by including `Firestore/core/src/model/model_fwd.h`.

Also, this is inconsistent with `key_field_not_in_filter.h` which neither makes the forward declarations nor includes the `model_fwd.h` header.